### DISCUSSION TITLE
hackmd / notion / medium or others

### DISCUSSION THREAD

**Discussion by kevin861222** [bol-edu/HLS-SOC-Discussions#56] [General]
Hi all.
In this course, the teacher encourages students to use Hackmd to record class notes and share them publicly. Is there a strict requirement to use Hackmd, or can other tools like Notion or Medium be used? THX~

**Comment by boledulab** [comment]
There is no strict requirement as to how to use Hackmd. If you have other , or better way to write the SOC study journal, you are welcome to share.

**Reply by wcc12120508** [reply]
THX
